index.php

```diff
@@ -23,35 +23,18 @@
 			<div class="cocktails_box">
 
                 <?php
+				//linking to DB
+				require './getDB_function.php';
 
-                // Accessing DB from PHP
-                require_once './getDB_function';
-                $db= getDb("db", "Cocktails", "root", "password");
-                $query=$db->prepare("SELECT `name`, `alcohol_base`, `taste_profile`, `ingredients`, `method`, `strength`, `served`  FROM `cocktails`;");
-                $query->execute();
-                $result = $query->fetchAll();
-
+				// to get the cocktails data from the db
+				require './getCocktails_function.php';
+				$result = getCocktails();
+				
                 //echoing results of db to website
-                foreach ($result as $cocktail) {
-                    echo '<div class="box1">';
-                    echo    '<h2>' . $cocktail['name'] .
-                            '</h2>';
-                    echo    '<ul>
-                                <li>' . $cocktail['alcohol_base'] .
-                                '</li>';
-                    echo        '<li>' . $cocktail['taste_profile'] .
-                                '</li>';
-                    echo        '<li>' . $cocktail['ingredients'] .
-                                '</li>';
-                    echo        '<li>' . $cocktail['method'] .
-                                '</li>';
-                    echo        '<li>' . $cocktail['strength'] .
-                                '</li>';
-                    echo        '<li>' . $cocktail['served'] .
-                                '</li>
-                            </ul>';
-                    echo '</div>';
-                }
+				require './outputCocktails_function.php';
```

Comment:
I would suggest that you move all of the require statements in this file to the top (before any HTML).

This doesn't really make any difference in terms of how the code works, but it does make it a bit easier to tell which files are required when you open the file without having to read through. Does that make sense?

Reply:
ok no problem will do